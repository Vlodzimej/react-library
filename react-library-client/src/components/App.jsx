import React, { useState } from 'react';
import 'babel-polyfill';
import { BookContext } from './context';
import Book from './book';
import BookList from './booklist';

export const App = () => {
    const [bookId, setBookId] = useState(0);
    return (
        <div>
            <BookContext.Provider value={[bookId, setBookId]}>
                <BookList />
                <Book />
            </BookContext.Provider>
        </div>
    );
};
export default App;
