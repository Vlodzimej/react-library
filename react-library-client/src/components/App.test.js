/* eslint-disable no-undef */
// App.test.js
import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';
test('Check render App component', () => {
    const component = renderer.create(<App />);
    let wrapper = component.toJSON();
    expect(wrapper).toMatchSnapshot();
});
