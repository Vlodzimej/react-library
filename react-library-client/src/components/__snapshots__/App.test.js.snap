// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check render App component 1`] = `
<div>
  <h1>
    My React App!!!
  </h1>
</div>
`;

exports[`Link changes the class when hovered 1`] = `
<div>
  <h1>
    My React App!!!
  </h1>
</div>
`;
